### PR TITLE
[wip] Merge Develop into Master

### PR DESCRIPTION
**Pull request by svenjens**

### CODE REVIEW

File: app/shared/services/environment.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+export const ENVIRONMENTS = {
```
Comment (svenjens) [inline app/shared/services/environment.js]: Checken waar we wat voor nodig hebben. Kan er iets weg?

File: app/containers/HomePage/messages.js [hunk 2]
```diff
@@ -8,6 +8,6 @@ import { defineMessages } from 'react-intl';
 export default defineMessages({
   header: {
     id: 'app.components.HomePage.header',
-    defaultMessage: 'This is HomePage component!',
+    defaultMessage: 'Vul het betreffende postcode en huisnummer in:',
```
Comment (svenjens) [inline app/containers/HomePage/messages.js]: Do we use this?

File: app/index.html [hunk 3]
```diff
@@ -1,42 +1,152 @@
-<!doctype html>
+<!DOCTYPE html>
```
Comment (svenjens) [inline app/index.html]: create static

File: Jenkinsfile [hunk 4]
```diff
@@ -94,3 +94,61 @@ if (BRANCH == "master") {
...
+            tryStep "deployment", {
+                build job: 'Subtask_Openstack_Playbook',
+                parameters: [
+                    [$class: 'StringParameterValue', name: 'INVENTORY', value: 'production'],
+                    [$class: 'StringParameterValue', name: 'PLAYBOOK', value: 'deploy-vergunningschecker.yml'],
+                ]
+            }
+        }
+    }
+}
```
Comment (robinpiets) [inline Jenkinsfile]: Let's make a plan when to build to `acceptance` and `production`. (With this code you can build production from the develop branch)

**Pull request merged**